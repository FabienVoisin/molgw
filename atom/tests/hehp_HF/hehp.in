HF                   # calculation type HF, LDA+GW, PBE+GW
1   1.0   0.0        # spin pol    charge magnetization
STO-3G               #6-31G # cc-pVTZ                  #          6-311ppGss               # basis set
10   0.80            # nscf, alpha_mixing
11                   # print_volume
2
2.    0.0000 0.0  0.0
1.    1.4632 0.0  0.0
