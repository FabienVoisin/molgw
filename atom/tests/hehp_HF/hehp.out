 
 Welcome to the fascinating world of ATOM
 
 
 before read
 inside read_input

 WARNING: calculation name: HF

 Summary of the input parameters 

  calculation type:   HF
             Natom:   2
         Electrons:   2.0000
            Charge:   1.0000
     Magnetization:   0.0000
         Basis set:   STO-3G                                                                                              
 Spin polarization:   1
         SCF steps:  10
            Mixing:   0.8000
 
 
 Element used for Z value:    He
 Element used for the basis:  He
 Basis type: STO-3G

 WARNING: basis file used: He_STO-3G
 
 open the basis set file He_STO-3G
 
 Element used for Z value:    H
 Element used for the basis:  H
 Basis type: STO-3G

 WARNING: basis file used: H_STO-3G
 
 open the basis set file H_STO-3G
 Total number of basis functions           2
 
 
 ======  print out a basis function ======
    contraction of N gaussians     3
              orbital momentum     s
                   centered in     0.000000      0.000000      0.000000
                   coefficient     1      0.154329
                   coefficient     2      0.535328
                   coefficient     3      0.444635
 
 
 *** output information of a cartesian gaussian function ***
              orbital momentum     s
          momentum composition     0   0   0
                   alpha coeff     9.753935
        center of the function     0.000000     0.000000     0.000000
 
 
 *** output information of a cartesian gaussian function ***
              orbital momentum     s
          momentum composition     0   0   0
                   alpha coeff     1.776691
        center of the function     0.000000     0.000000     0.000000
 
 
 *** output information of a cartesian gaussian function ***
              orbital momentum     s
          momentum composition     0   0   0
                   alpha coeff     0.480844
        center of the function     0.000000     0.000000     0.000000
 
 ====== end of basis function ======
 
 
 ======  print out a basis function ======
    contraction of N gaussians     3
              orbital momentum     s
                   centered in     1.463200      0.000000      0.000000
                   coefficient     1      0.154329
                   coefficient     2      0.535328
                   coefficient     3      0.444635
 
 
 *** output information of a cartesian gaussian function ***
              orbital momentum     s
          momentum composition     0   0   0
                   alpha coeff     3.425250
        center of the function     1.463200     0.000000     0.000000
 
 
 *** output information of a cartesian gaussian function ***
              orbital momentum     s
          momentum composition     0   0   0
                   alpha coeff     0.623913
        center of the function     1.463200     0.000000     0.000000
 
 
 *** output information of a cartesian gaussian function ***
              orbital momentum     s
          momentum composition     0   0   0
                   alpha coeff     0.168856
        center of the function     1.463200     0.000000     0.000000
 
 ====== end of basis function ======
 
 Basis set is ready and fit

 === overlap matrix S ===
   1      1.00000      0.45077
   2      0.45077      1.00000
 
 

 Allocating the ERI array:    0.000 [Mb]
 success

 Calculate and store all the Electron Repulsion Integrals (ERI)
 number of shells           6
 Done.
 

 === occupations ===
   1      2.00000
   2      0.00000
 

 === 1st density matrix P ===
   1      2.00000      0.00000
   2      0.00000      0.00000
 
 

 === kinetic energy contribution ===
   1      2.16431      0.16701
   2      0.16701      0.76003
 
 
call boys   0      0.0000000000      1.0000000000               NaN
call boys   0      0.0000000000      1.0000000000               NaN
call boys   0      0.0000000000      1.0000000000               NaN
call boys   0      0.0000000000      1.0000000000               NaN
call boys   0      0.0000000000      1.0000000000               NaN
call boys   0      0.0000000000      1.0000000000               NaN
call boys   0      0.0000000000      1.0000000000               NaN
call boys   0      0.0000000000      1.0000000000               NaN
call boys   0      0.0000000000      1.0000000000               NaN
call boys   0      1.9059144106      0.6092688072      0.6092688072
call boys   0      0.0803060317      0.9738640878      0.9738640878
call boys   0      0.0061518615      0.9979531585      0.9979531585
call boys   0      4.8286586424      0.4025425939      0.4025425939
call boys   0      0.3471641818      0.8953982905      0.8953982905
call boys   0      0.0313760761      0.9896390228      0.9896390228
call boys   0      6.4305661667      0.3493611549      0.3493611549
call boys   0      0.7543772357      0.7965601315      0.7965601315
call boys   0      0.0939566042      0.9695445232      0.9695445232
call boys   0      1.9059144106      0.6092688072      0.6092688072
call boys   0      4.8286586424      0.4025425939      0.4025425939
call boys   0      6.4305661667      0.3493611549      0.3493611549
call boys   0      0.0803060317      0.9738640878      0.9738640878
call boys   0      0.3471641818      0.8953982905      0.8953982905
call boys   0      0.7543772357      0.7965601315      0.7965601315
call boys   0      0.0061518615      0.9979531585      0.9979531585
call boys   0      0.0313760761      0.9896390228      0.9896390228
call boys   0      0.0939566042      0.9695445232      0.9695445232
call boys   0     14.6666070211      0.2314089152      0.2314089152
call boys   0      8.6690726933      0.3009851241      0.3009851241
call boys   0      7.6948164797      0.3194536241      0.3194536241
call boys   0      8.6690726933      0.3009851241      0.3009851241
call boys   0      2.6715383655      0.5309254494      0.5309254494
call boys   0      1.6972821519      0.6357528052      0.6357528052
call boys   0      7.6948164797      0.3194536241      0.3194536241
call boys   0      1.6972821519      0.6357528052      0.6357528052
call boys   0      0.7230259383      0.8033981449      0.8033981449
call boys   0     41.7654552771      0.1371312171      0.1371312171
call boys   0     24.6865419823      0.1783671245      0.1783671245
call boys   0     21.9121932814      0.1893224063      0.1893224063
call boys   0     24.6865419823      0.1783671245      0.1783671245
call boys   0      7.6076286874      0.3212762523      0.3212762523
call boys   0      4.8332799866      0.4023539176      0.4023539176
call boys   0     21.9121932814      0.1893224063      0.1893224063
call boys   0      4.8332799866      0.4023539176      0.4023539176
call boys   0      2.0589312857      0.5914163689      0.5914163689
call boys   0     15.4553385386      0.2254268631      0.2254268631
call boys   0     19.6272644875      0.2000391742      0.2000391742
call boys   0     20.5273665309      0.1956042659      0.1956042659
call boys   0      1.2991694755      0.6943449384      0.6943449384
call boys   0      2.8152093427      0.5188660044      0.5188660044
call boys   0      3.4736774507      0.4715081448      0.4715081448
call boys   0      0.1267282990      0.9593159509      0.9593159509
call boys   0      0.4480736958      0.8687509558      0.8687509558
call boys   0      0.7619092779      0.7949347333      0.7949347333
call boys   0     15.4553385386      0.2254268631      0.2254268631
call boys   0      1.2991694755      0.6943449384      0.6943449384
call boys   0      0.1267282990      0.9593159509      0.9593159509
call boys   0     19.6272644875      0.2000391742      0.2000391742
call boys   0      2.8152093427      0.5188660044      0.5188660044
call boys   0      0.4480736958      0.8687509558      0.8687509558
call boys   0     20.5273665309      0.1956042659      0.1956042659
call boys   0      3.4736774507      0.4715081448      0.4715081448
call boys   0      0.7619092779      0.7949347333      0.7949347333
call boys   0      0.0000000000      1.0000000000               NaN
call boys   0      0.0000000000      1.0000000000               NaN
call boys   0      0.0000000000      1.0000000000               NaN
call boys   0      0.0000000000      1.0000000000               NaN
call boys   0      0.0000000000      1.0000000000               NaN
call boys   0      0.0000000000      1.0000000000               NaN
call boys   0      0.0000000000      1.0000000000               NaN
call boys   0      0.0000000000      1.0000000000               NaN
call boys   0      0.0000000000      1.0000000000               NaN

 === nucleus contribution ===
   1     -4.81706     -1.51422
   2     -1.51422     -2.49186
 
 

-------------------------------------------
 *** SCF cycle No:    1


 === Hartree contribution ===
   1      2.61430      0.87456
   2      0.87456      1.21141
 
 

 === Exchange contribution ===
   1     -1.30715     -0.43728
   2     -0.43728     -0.17727
 
 

 === Total Hamiltonian ===
   1     -1.34559     -0.90993
   2     -0.90993     -0.69769
 
 
 Diagonalization for spin polarization           1

 === Energies ===
   #       [Ha]         [eV]      
   1     -1.43808    -39.13215
   2     -0.09673     -2.63216
 

 === C coefficients ===
   1     -0.83231      0.74985
   2     -0.29417     -1.08096
 
 

 === density matrix P ===
   1      1.38547      0.48968
   2      0.48968      0.17307
 
 
 convergence criterium on the density matrix  0.941896314493580     
 
    Nucleus-Nucleus [Ha]:     1.3668671405
    Kinetic Energy  [Ha]:     4.3286250819
    Nucleus Energy  [Ha]:    -9.6341145695
    Hartree Energy  [Ha]:     2.6143032312
    Exchange Energy [Ha]:    -1.3071516156
 
    Total Energy    [Ha]:    -2.6314707314
 

-------------------------------------------
 *** SCF cycle No:    2


 === Hartree contribution ===
   1      2.39813      0.84164
   2      0.84164      1.26516
 
 

 === Exchange contribution ===
   1     -1.16941     -0.50474
   2     -0.50474     -0.30946
 
 

 === Total Hamiltonian ===
   1     -1.42402     -1.01031
   2     -1.01031     -0.77613
 
 
 Diagonalization for spin polarization           1

 === Energies ===
   #       [Ha]         [eV]      
   1     -1.54967    -42.16857
   2     -0.06843     -1.86221
 

 === C coefficients ===
   1     -0.80957      0.77434
   2     -0.32628     -1.07170
 
 

 === density matrix P ===
   1      1.31080      0.52829
   2      0.52829      0.21292
 
 
 convergence criterium on the density matrix  0.286137531140692     
 
    Nucleus-Nucleus [Ha]:     1.3668671405
    Kinetic Energy  [Ha]:     3.5006723254
    Nucleus Energy  [Ha]:    -8.7972906202
    Hartree Energy  [Ha]:     2.2259273357
    Exchange Energy [Ha]:    -1.1010997953
 
    Total Energy    [Ha]:    -2.8049236139
 

-------------------------------------------
 *** SCF cycle No:    3


 === Hartree contribution ===
   1      2.32315      0.82982
   2      0.82982      1.28369
 
 

 === Exchange contribution ===
   1     -1.11915     -0.52223
   2     -0.52223     -0.35258
 
 

 === Total Hamiltonian ===
   1     -1.44875     -1.03961
   2     -1.03961     -0.80072
 
 
 Diagonalization for spin polarization           1

 === Energies ===
   #       [Ha]         [eV]      
   1     -1.58406    -43.10440
   2     -0.06279     -1.70852
 

 === C coefficients ===
   1     -0.80389      0.78024
   2     -0.33411     -1.06929
 
 

 === density matrix P ===
   1      1.29246      0.53717
   2      0.53717      0.22325
 
 
 convergence criterium on the density matrix  8.124966360480752E-002
 
    Nucleus-Nucleus [Ha]:     1.3668671405
    Kinetic Energy  [Ha]:     3.2403489994
    Nucleus Energy  [Ha]:    -8.5151907669
    Hartree Energy  [Ha]:     2.1113213467
    Exchange Energy [Ha]:    -1.0521445112
 
    Total Energy    [Ha]:    -2.8487977915
 

-------------------------------------------
 *** SCF cycle No:    4


 === Hartree contribution ===
   1      2.30020      0.82614
   2      0.82614      1.28934
 
 

 === Exchange contribution ===
   1     -1.10335     -0.52659
   2     -0.52659     -0.36532
 
 

 === Total Hamiltonian ===
   1     -1.45590     -1.04765
   2     -1.04765     -0.80781
 
 
 Diagonalization for spin polarization           1

 === Energies ===
   #       [Ha]         [eV]      
   1     -1.59380    -43.36942
   2     -0.06182     -1.68225
 

 === C coefficients ===
   1     -0.80243      0.78174
   2     -0.33610     -1.06866
 
 

 === density matrix P ===
   1      1.28778      0.53940
   2      0.53940      0.22593
 
 
 convergence criterium on the density matrix  2.243029071747944E-002
 
    Nucleus-Nucleus [Ha]:     1.3668671405
    Kinetic Energy  [Ha]:     3.1651943783
    Nucleus Energy  [Ha]:    -8.4302165257
    Hartree Energy  [Ha]:     2.0782389644
    Exchange Energy [Ha]:    -1.0383213536
 
    Total Energy    [Ha]:    -2.8582373961
 

-------------------------------------------
 *** SCF cycle No:    5


 === Hartree contribution ===
   1      2.29356      0.82507
   2      0.82507      1.29097
 
 

 === Exchange contribution ===
   1     -1.09871     -0.52768
   2     -0.52768     -0.36893
 
 

 === Total Hamiltonian ===
   1     -1.45790     -1.04981
   2     -1.04981     -0.80978
 
 
 Diagonalization for spin polarization           1

 === Energies ===
   #       [Ha]         [eV]      
   1     -1.59647    -43.44213
   2     -0.06168     -1.67842
 

 === C coefficients ===
   1     -0.80205      0.78213
   2     -0.33662     -1.06850
 
 

 === density matrix P ===
   1      1.28657      0.53997
   2      0.53997      0.22663
 
 
 convergence criterium on the density matrix  6.092788757356631E-003
 
    Nucleus-Nucleus [Ha]:     1.3668671405
    Kinetic Energy  [Ha]:     3.1442722300
    Nucleus Energy  [Ha]:    -8.4058988820
    Hartree Energy  [Ha]:     2.0688428150
    Exchange Energy [Ha]:    -1.0342544078
 
    Total Energy    [Ha]:    -2.8601711043
 

-------------------------------------------
 *** SCF cycle No:    6


 === Hartree contribution ===
   1      2.29170      0.82476
   2      0.82476      1.29143
 
 

 === Exchange contribution ===
   1     -1.09740     -0.52795
   2     -0.52795     -0.36992
 
 

 === Total Hamiltonian ===
   1     -1.45844     -1.05039
   2     -1.05039     -0.81032
 
 
 Diagonalization for spin polarization           1

 === Energies ===
   #       [Ha]         [eV]      
   1     -1.59719    -43.46176
   2     -0.06167     -1.67805
 

 === C coefficients ===
   1     -0.80195      0.78223
   2     -0.33675     -1.06846
 
 

 === density matrix P ===
   1      1.28625      0.54012
   2      0.54012      0.22681
 
 
 convergence criterium on the density matrix  1.636882533901026E-003
 
    Nucleus-Nucleus [Ha]:     1.3668671405
    Kinetic Energy  [Ha]:     3.1385638814
    Nucleus Energy  [Ha]:    -8.3991386380
    Hartree Energy  [Ha]:     2.0662254121
    Exchange Energy [Ha]:    -1.0330787548
 
    Total Energy    [Ha]:    -2.8605609587
 

-------------------------------------------
 *** SCF cycle No:    7


 === Hartree contribution ===
   1      2.29120      0.82468
   2      0.82468      1.29155
 
 

 === Exchange contribution ===
   1     -1.09704     -0.52802
   2     -0.52802     -0.37019
 
 

 === Total Hamiltonian ===
   1     -1.45859     -1.05054
   2     -1.05054     -0.81046
 
 
 Diagonalization for spin polarization           1

 === Energies ===
   #       [Ha]         [eV]      
   1     -1.59738    -43.46700
   2     -0.06167     -1.67807
 

 === C coefficients ===
   1     -0.80193      0.78226
   2     -0.33679     -1.06845
 
 

 === density matrix P ===
   1      1.28617      0.54016
   2      0.54016      0.22685
 
 
 convergence criterium on the density matrix  4.362610400844625E-004
 
    Nucleus-Nucleus [Ha]:     1.3668671405
    Kinetic Energy  [Ha]:     3.1370265431
    Nucleus Energy  [Ha]:    -8.3972939612
    Hartree Energy  [Ha]:     2.0655086134
    Exchange Energy [Ha]:    -1.0327474763
 
    Total Energy    [Ha]:    -2.8606391405
 

-------------------------------------------
 *** SCF cycle No:    8


 === Hartree contribution ===
   1      2.29106      0.82466
   2      0.82466      1.29159
 
 

 === Exchange contribution ===
   1     -1.09694     -0.52803
   2     -0.52803     -0.37026
 
 

 === Total Hamiltonian ===
   1     -1.45863     -1.05058
   2     -1.05058     -0.81050
 
 
 Diagonalization for spin polarization           1

 === Energies ===
   #       [Ha]         [eV]      
   1     -1.59743    -43.46840
   2     -0.06167     -1.67811
 

 === C coefficients ===
   1     -0.80192      0.78226
   2     -0.33680     -1.06845
 
 

 === density matrix P ===
   1      1.28615      0.54017
   2      0.54017      0.22687
 
 
 convergence criterium on the density matrix  1.155815271536334E-004
 
    Nucleus-Nucleus [Ha]:     1.3668671405
    Kinetic Energy  [Ha]:     3.1366162438
    Nucleus Energy  [Ha]:    -8.3967969838
    Hartree Energy  [Ha]:     2.0653148789
    Exchange Energy [Ha]:    -1.0326560705
 
    Total Energy    [Ha]:    -2.8606547911
 

-------------------------------------------
 *** SCF cycle No:    9


 === Hartree contribution ===
   1      2.29102      0.82465
   2      0.82465      1.29160
 
 

 === Exchange contribution ===
   1     -1.09691     -0.52804
   2     -0.52804     -0.37028
 
 

 === Total Hamiltonian ===
   1     -1.45864     -1.05059
   2     -1.05059     -0.81051
 
 
 Diagonalization for spin polarization           1

 === Energies ===
   #       [Ha]         [eV]      
   1     -1.59745    -43.46876
   2     -0.06167     -1.67812
 

 === C coefficients ===
   1     -0.80192      0.78226
   2     -0.33680     -1.06844
 
 

 === density matrix P ===
   1      1.28614      0.54017
   2      0.54017      0.22687
 
 
 convergence criterium on the density matrix  3.048474069411968E-005
 
    Nucleus-Nucleus [Ha]:     1.3668671405
    Kinetic Energy  [Ha]:     3.1365074519
    Nucleus Energy  [Ha]:    -8.3966643018
    Hartree Energy  [Ha]:     2.0652630266
    Exchange Energy [Ha]:    -1.0326312393
 
    Total Energy    [Ha]:    -2.8606579222
 

-------------------------------------------
 *** SCF cycle No:   10


 === Hartree contribution ===
   1      2.29101      0.82465
   2      0.82465      1.29160
 
 

 === Exchange contribution ===
   1     -1.09691     -0.52804
   2     -0.52804     -0.37028
 
 

 === Total Hamiltonian ===
   1     -1.45864     -1.05059
   2     -1.05059     -0.81051
 
 
 Diagonalization for spin polarization           1

 === Energies ===
   #       [Ha]         [eV]      
   1     -1.59745    -43.46886
   2     -0.06167     -1.67812
 

 === C coefficients ===
   1     -0.80192      0.78226
   2     -0.33680     -1.06844
 
 

 === density matrix P ===
   1      1.28614      0.54017
   2      0.54017      0.22687
 
 
 convergence criterium on the density matrix  8.013117238528037E-006
 
    Nucleus-Nucleus [Ha]:     1.3668671405
    Kinetic Energy  [Ha]:     3.1364787437
    Nucleus Energy  [Ha]:    -8.3966291116
    Hartree Energy  [Ha]:     2.0652492482
    Exchange Energy [Ha]:    -1.0326245693
 
    Total Energy    [Ha]:    -2.8606585484
 
 ==================================================
 The SCF loop ends here
 ==================================================
 

                 --- Timings in (s) and # of calls ---
                    Total time          0.02
 
         4 gaussians integrals          0.01
 
                       Hartree          0.00        10
                      Exchange          0.00        10
                        DFT xc          0.00         0
 
           ERI basis transform          0.00         0
 
         Total GW polarization          0.00         0
            Diago 2 particle H          0.00         0
           Invert 2 particle S          0.00         0
 
                GW self-energy          0.00         0
 
                    MP2 energy          0.00         0
               MP2 self-energy          0.00         0
                 ----------------------

 SUMMARY of all the WARNINGS issued along the run:

 1.     calculation name: HF                                                                                
 2.     basis file used: He_STO-3G                                                                          
 3.     basis file used: H_STO-3G                                                                           



 This is the end

